chart/templates/deployment.yaml

```
@@ -58,10 +60,12 @@ spec:
                   - {{ template "rancher.fullname" . }}
               topologyKey: {{ .Values.topologyKey | default "kubernetes.io/hostname" }}
 {{- end }}
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms: {{ include "linux-node-selector-terms" . | nindent 14 }}
-      tolerations: {{ include "linux-node-tolerations" . | nindent 8 }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
```

I don't think you need to create a new line when nindenting.

```suggestion
      nodeSelector: {{ toYaml . | nindent 8 }}
```